src/test/java/org/interledger/FulfillmentTest.java

```diff
@@ -0,0 +1,49 @@
+package org.interledger;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import java.util.Base64;
+
+public class FulfillmentTest {
+
+  private static byte[] TEST_PREIMAGE = new byte[32];
+  private static Condition TEST_CONDITION = Condition.builder()
+      .hash(Base64.getUrlDecoder().decode("Zmh6rfhivXdsj8GLjp-OIAiXFIVu4jOzkCpZHQ1fKSU")).build();
```

Comment:
It might be preferable to add a 2nd additional test that generates 256 bits of random byte data here for the preimage so that each time the test runs, it validates a different preimage? Then we would have some static coverage (what's there now) as well as on-going assurances across different hash values.